src/T2DMap.cpp

```diff
@@ -2513,389 +2513,128 @@ void T2DMap::mouseReleaseEvent(QMouseEvent* e)
         mMultiRect = QRect(0, 0, 0, 0);
         update();
```

Comment:
As only one button can be released at a time I think that you can/should put a `return;` under this line as there is no need to proceed any further in the handler...

Reply:
done